modules/account-lib/src/coin/dot/singletonRegistry.ts

```diff
@@ -0,0 +1,24 @@
+import { TypeRegistry } from '@substrate/txwrapper-core/lib/types';
+import { getRegistry } from '@substrate/txwrapper-polkadot';
+import { PolkadotSpecNameType } from '@bitgo/statics';
+import { Metadata } from '@polkadot/types/metadata/Metadata';
+import { Material } from './iface';
+
+export class SingletonRegistry {
+  private static instance: TypeRegistry;
+  private static material: Material;
+  private static metadata: Metadata;
```

Comment:
`metadata` doesn't seem to be read? Or did I miss something. Where do we need this for?:
```ts
'metadata' is declared but its value is never read
```

Reply:
You are right, we tried a different approach that did not work and forgot to delete that variable.